src/store/Configuration/mutations.ts

```diff
@@ -0,0 +1,38 @@
+import { ConfigurationStateIF } from '@/interfaces'
+import Vue from 'vue'
+import axios from '@/axios-auth'
+
+export default {
+  setConfiguration (state: ConfigurationStateIF, data: any) {
+    state.configuration = data
+  },
+  /** Use this mutator to set a specific attribute in unit testing.
+   * Like this: store.commit('setTestConfiguration', ['AUTH_WEB_URL', 'https://auth.web.url/'])
+   * */
+  setTestConfiguration (state: ConfigurationStateIF, payload: any) {
```

Comment:
If you have time, please create a type for payload.

Does it have to be an array, or is an object OK? Eg,
```
store.commit('setTestConfiguration', { key: 'AUTH_WEB_URL', val: 'https://auth.web.url/' })
```

Reply:
I like your suggestion of a creating an object way better!  I'm on it ...